src/app/static/src/app/components/projects/ProjectHistory.vue

```diff
@@ -4,7 +4,7 @@
         <div id="headers" class="row font-weight-bold pt-2">
             <div class="col-md-1 header-cell"></div>
             <div class="col-md-3 header-cell" v-translate="'projectName'"></div>
-            <div class="col-md-1 header-cell">Versions</div>
+            <div class="col-md-1 header-cell" v-translate="'versions'"></div>
```

Comment:
'versions' now needs to be translated because, unlike French, the Portuguese for this word is different from English. Also, because the Portugues for 'versions' is not simply the same as for 'version' with an s on the end, we now need to translate for singular and plural separately.